CSharp/NeoMatrix.Validation/Validators/ValidatorUtility.cs

```diff
@@ -64,5 +85,32 @@ private static ValidateResult<bool> ValidateByKeyOnly(JsonDocument doc, string w
             }
             return new ValidateResult<bool>() { Result = true };
         }
+
+        private static ValidateResult<bool> ValidateByKeyValuePair(JsonDocument doc, string pair)
+        {
+            var pairArray = pair.Split(":").Select(w => w.Replace(";", "")).ToArray();

```

Comment:
string.Replace() is not efficient & needn't use Linq here. Should split by the char ';' then split by ':', this way makes the following error check easy.
The error check: should deal with problems of 'word', like `param1:;param2:value2` or `param1;param2:value2`. If the value1 is missing, skip/ignore param1 and use param2 only.

Reply:
Sounds great. Thanks for suggestion!